etc/templates/profile.template

```diff
@@ -195,6 +195,7 @@ include globals.local
 ##x11 none
 
 #disable-mnt
+##keep-dev-ntsync
```

Comment:
Placement here?

Reply:
> Placement here?

I agree that it does not seem like the best place, though putting it in the
`no3d` section (for example) does not seem much better either so I just left it
as is.  But I can move it somewhere else if you want.

I plan to create a PR after this to create an "allow" section above the `no3d`
etc section with only allow commands (like the second section in
noprofile.profile) and move the relevant commands there.